src/meetings/2018-03-13/index.md

```diff
@@ -1,44 +1,9 @@
 ---
-meeting:
-  date: 2018-03-13
-  time:
-    start: 6:00pm
-    end: 7:00pm
-host:
-  location: Gravitate, 317 6th Ave, Suite 10
-  city: Des Moines
-  state: IA
-  zip: 50309
-sponsor:
-  name: Robert Half
-  site: https://www.roberthalf.com/ia-des-moines
-talk: 
-  title: An Introduction to Transducers
-  tags: Functional Programming
-  speaker: Eric Ponto
-  twitter: ericponto
-  slides: http://slides.ericponto.com/dsmjs-intro-to-transducers/
+date: 2018-03-13
+time:
+  start: 6:00pm
+  end: 7:00pm
+host: Gravitate
+sponsor: Robert Half
+talk: An Introduction to Transducers
```

Comment:
Should we add a `slides` and or `source_code` key/value for if people already have them hosted somewhere?

Reply:
while the components don't support showing them yet, i do intend to bring that data over and get it into the components before we'd make the switch. for example, the data is at least in this talk for `slides`.  somewhat related, i plan to add github to the speaker data so that people can easily link to their profile there in addition to `twitter`